.circleci/config.yml

```diff
@@ -116,7 +116,7 @@ jobs:
             equal: [ master, << pipeline.git.branch >> ]
           steps:
             - deploy:
-                # Publishing should be the first step since if it fails, we should NOT proceed opening backport PR/GitHub release etc.
+                # Publishing should be the first step since if it fails, we should NOT proceed with opening backport PR/GitHub release, etc.
                 name: Publish plugin to Jetbrains Marketplace
                 command: ./gradlew --max-workers=2 publishPlugin
```

Comment:
While you're here... pls also remove `--max-workers=2`, this should no longer be necessary (I hope!)